pyccl/boltzmann.py

```diff
@@ -43,6 +46,14 @@ def get_camb_pk_lin(cosmo):
         "You must have the `camb` python package "
         "installed to run CCL with CAMB!")
 
+    # Get extra CAMB parameters that were specified
+    extra_camb_params = {}
+    try:
+        extra_camb_params = \
+            cosmo._params_init_kwargs["extra_parameters"]["camb"]
```

Comment:
Anything with an underscore (`_`) in front of it is considered private. So in this case, the Cosmology class needs to add a public API that exposes this information. We should not be calling things that are considered private, even from our own code.

Reply:
Now exposed through `__getitem__`, like the other parameters.

Reply:
Looks good to me!